plugins/node/opentelemetry-instrumentation-aws-sdk/test/MessageAttributes.test.ts

```diff
@@ -79,12 +82,14 @@ describe('MessageAttributes', () => {
         key5: { DataType: 'String', StringValue: 'value5' },
       };
 
-      expect(Object.keys(contextAttributes).length).toBe(5);
       injectPropagationContext(contextAttributes);
-      expect(Object.keys(contextAttributes).length).toBeGreaterThan(5);
+      // @ts-expect-error Property 'calledOnce' does not exist on type
+      expect(propagation.inject.calledOnce).toBe(true);
+      sinon.restore();
```

Comment:
Was there any conclusion why the tests started failing for the `Object.keys(contextAttributes).length > 5` expectation? 

Otherwise, the rest of the changes looks good to me.

Reply:
I don't know. Instead of checking for edits done to the object, I replaced with check for calls which a unit test should do.

Reply:
Your update is for the better.